### PR TITLE
Add support for borders on wizardlets and AlertMessageBorder

### PR DESCRIPTION
**Pull request by sarthak-marwaha**

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]
I'm good with this. Thanks.

**Review by adamfox-nanopay — APPROVED** [review]

File: src/foam/u2/borders/AlertMessageBorder.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      adapt: function(_, n) {
+        var lastIndex = n.lastIndexOf('.');
+        var classObj = foam.lookup(n.substring(0, lastIndex));
+        return classObj[n.substring(lastIndex + 1)];
+      }
+    },
+    'futureContent_'
+  ],
+  methods: [
+    function init() {
```
Comment (kgrgreer) [inline src/foam/u2/borders/AlertMessageBorder.js]: Could this be render()?

File: src/foam/u2/borders/AlertMessageBorder.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      flex-direction: column;
+      gap: 1rem;
+    }
+  `,
+  properties: [
+    {
+      class: 'String',
+      name: 'source',
+      adapt: function(_, n) {
+        var lastIndex = n.lastIndexOf('.');
```
Comment (kgrgreer) [inline src/foam/u2/borders/AlertMessageBorder.js:34]: Could you comment what this is doing?

**Comment by kgrgreer** [comment]
Please add a sample usage to the docs, something like:
        border: {
          class: "foam.u2.borders.AlertMessageBorder",
          source: 'net.nanopay.partner.nbp.NBPTransferQuoteRequest.MAINTENANCE_MESSAGE'
        },
Thx

**Pull request merged**